### ISSUE TITLE
Timbermaw Hold rep

### ISSUE THREAD

**Issue by Meachy** [dalaranwow/dalaran-wow#5632]
**Current behaviour**: When in Felwood farming timbermaw hold rep, I do not gain any rep when my friend kills the furbolgs. We are in the same party and I tried to gain rep as ghost, alive, tagging the mobs, none of it worked. This happened on my level 1 "Bipi". I do have the Timbermaw Hold faction in my reputation pane, but it hasn't budged from hated 2500/3000 while my high level friend has been killing them and gaining full rep.

**Expected behaviour**: I should receive rep when my friend kill the furbolgs, which i am not receiving.

**Steps to reproduce the problem**:

1. bring low level to felwood grouped with high level
2. furbolgs are killed by high level
3. low level has no rep gain

**Include proofs for this behaviour**
http://wowwiki.wikia.com/wiki/Timbermaw_Hold

**Comment by galgnu** [comment]
Pretty sure you have to do the first couple of quest in order to be able to get reputation. Those quest of course requires a higher level.

**Comment by Zarjofgt** [comment]
No, you can get rep without doing any quests or other requirments. Lvl 1 twinks in wotlk had the medicine bag.

**Comment by maciejpk** [comment]
I just came upon this issue as well, it's kinda terrible because I cannot get offhand for my lvl 1 twink, so +1
